### ISSUE TITLE
NullPointerException in Binarylog position comparing

### ISSUE THREAD

**Issue by theikkila** [zendesk/maxwell#449]
Maxwell crashes at async bootstrap phase with NullPointerException. I'm not able to provide the exact data for creating the same situation again but here is something. I hope that somebody with same issue could reproduce the error and fill information about this issue.

### Stacktrace
```
Oct 07 12:37:41 maxwell maxwell[2496]: java.lang.NullPointerException
Oct 07 12:37:41 maxwell maxwell[2496]: at java.lang.String.compareTo(String.java:1155)
Oct 07 12:37:41 maxwell maxwell[2496]: at com.zendesk.maxwell.BinlogPosition.newerThan(BinlogPosition.java:55)
Oct 07 12:37:41 maxwell maxwell[2496]: at com.zendesk.maxwell.bootstrap.AsynchronousBootstrapper.replaySkippedRows(AsynchronousBootstrapper.java:131)
Oct 07 12:37:41 maxwell maxwell[2496]: at com.zendesk.maxwell.bootstrap.AsynchronousBootstrapper.completeBootstrap(AsynchronousBootstrapper.java:97)
Oct 07 12:37:41 maxwell maxwell[2496]: at com.zendesk.maxwell.bootstrap.AsynchronousBootstrapper.work(AsynchronousBootstrapper.java:172)
Oct 07 12:37:41 maxwell maxwell[2496]: at com.zendesk.maxwell.MaxwellReplicator.work(MaxwellReplicator.java:162)
Oct 07 12:37:41 maxwell maxwell[2496]: at com.zendesk.maxwell.RunLoopProcess.runLoop(RunLoopProcess.java:35)
Oct 07 12:37:41 maxwell maxwell[2496]: at com.zendesk.maxwell.Maxwell.start(Maxwell.java:163)
Oct 07 12:37:41 maxwell maxwell[2496]: at com.zendesk.maxwell.Maxwell.main(Maxwell.java:180)
```

**Comment by bmichaud** [comment]
If this [https://github.com/zendesk/maxwell/issues/443](url) is the same issue, then it is already fixed. :)

**Comment by osheroff** [comment]
yeah this is a dupe and is fixed in v1.3.0

**Comment by theikkila** [comment]
Perfect! Thanks guys!
